webapp/src/containers/wordpress/wordpressFeeds/index.js

```diff
@@ -0,0 +1,72 @@
+/*eslint-disable no-unused-vars*/
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import fetchData from '../../../lib/fetchData';
+import { fetchWp, fetchWpSuccess, fetchWpFailure } from '../../../actions/wp';
+import { selectWp } from '../../../selectors/wpSelectors';
+
+import WordpressNews from './wordpressNews';
+import WordpressPosts from './wordpressPosts';
+
+import{ WP_POST_BASE_URL,WP_POST_URL } from '../../../constants';
+
+
+class WordpressFeeds extends Component {
+  constructor(props){
+    super(props);
+  }
+  componentDidMount() {
+    this.getData(this.getCurrentRoute(this.props));
+  }
+  componentWillUpdate(nextProps, nextState){
+    if(this.getCurrentRoute(this.props) != this.getCurrentRoute(nextProps)){
+      this.getData(this.getCurrentRoute(nextProps));
+    }
+  }
+  getCurrentRoute(props){
+    return props.params.postId;
+  }
+  getData(currentRoute){
+    this.props.dispatch(fetchWp());
+    let homeUrl='';
+    if(this.props.params.postId==='news')homeUrl=WP_POST_BASE_URL;
```

Comment:
I thought it was simpler to differentiate the two components  at the render function level of wordpressFeed main component since the two fetch similar data the difference being at the display level.